### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (a[title="Edit\ Article\ \(fr-fr\)"])

### ISSUE THREAD

**Issue by brianteeman** [joomla/backend-template#407]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Articles - extra files - Administration](http://localhost/back/administrator/index.php?option=com_content&view=articles)
#### Element path
a[title="Edit\ Article\ \(fr-fr\)"]
#### Snippet
`<a class="hasTooltip" href="/back/administrator/index.php?option=com_content&amp;task=article.edit&amp;id=2" title="Edit Article (fr-fr)">
												Article (fr-fr)</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.42 (foreground color: #2077dd, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

#### Note
There are other contrast issues but I "think" fixing this one will fix the others
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by brianteeman** [comment]
https://webaim.org/resources/contrastchecker/?fcolor=2077DD&bcolor=FFFFFF
